### PR TITLE
GraphQL Endpoints

### PR DESCRIPTION
**Pull request by janbaykara**

### CODE REVIEW

File: vent_your_rent/api/engine/nationbuilder.py [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    return event
+
+# def get_event_attendees_by_id(id: str):
+#     attendees = nb_get('get', f'sites/{nation_slug}/pages/events/{id}/rsvps')
+#     return attendees
+
+
+# def create_rsvp_by_event_id(event_id, person):
+    # rsvp = nb_request(f'/ sites/{nation_slug}/pages/events/{event_id}/rsvps')
+    # TODO: update custom_fields['rsvp_count'] = Person.thisperson(person).count_rsvps()
```
Comment (conatus) [inline vent_your_rent/api/engine/nationbuilder.py]: Remove commented out code.

File: vent_your_rent/api/engine/nationbuilder.py [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+#     return
+
+
+# def get_neighbours(postcode):
+#     '''
+#     /people/search
+#     '''
+#     '''
+#     /people/nearby
+#     '''
```
Comment (conatus) [inline vent_your_rent/api/engine/nationbuilder.py]: Remove unrequired commented out code.

File: vent_your_rent/api/engine/services.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+from vent_your_rent.api.helpers.proxy import proxy_request
+import json
+import os
+import urllib.parse
+
+def autocomplete(request, search):
+    qs = urllib.parse.urlencode({
+        "key": os.getenv('GOOGLE_MAPS_API_KEY'),
+        "components": 'country:' + os.getenv('CCTLD'),
+        # "type": "address",
```
Comment (conatus) [inline vent_your_rent/api/engine/services.py]: Sorry to beat the drum on this, but the same deal with this commenting out.

File: vent_your_rent/api/helpers/cache.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    results.update(resolved_items)
+
+    return {
+        key_from_cache_key(key): value
+        for (key, value)
+        in results.items()
+    }
+
+def cache_resolve(key, resolve, bucket, cache_type='default', timeout_seconds=None):
+    # print('lookup', [bucket, key])
```
Comment (conatus) [inline vent_your_rent/api/helpers/cache.py]: Remove debugging code.

File: vent_your_rent/api/engine/nationbuilder.py (hunk 1 above)
Comment (janbaykara) [reply vent_your_rent/api/engine/nationbuilder.py]: This is all taken from momentum. Deleted this file for now, til we do this properly.

**Comment by janbaykara** [comment]
@conatus added admin panel and real data. Please double confirm.

**Comment by conatus** [comment]
@janbaykara Will do!

File: vent_your_rent/api/helpers/cache.py [hunk 5]
```diff
@@ -0,0 +1,70 @@
+from django.core.cache import caches
+from threading import Timer
+
+# Just caching everything in a hashmap because serialization overhead of django-cache
```
Comment (conatus) [inline vent_your_rent/api/helpers/cache.py]: Comment slightly badly spelt.

**Pull request merged**